examples/dice/README.md

```diff
@@ -0,0 +1,30 @@
+# Dice example
+
+Instructions on how to run instrumented and uninstrumented examples.
```

Comment:
Can you write a few words that this is the foundation for https://opentelemetry.io/docs/languages/go/getting-started/ ?